apps/redi-talent-pool/src/components/organisms/company-profile-editables/EditableContact.tsx

```diff
@@ -75,6 +75,14 @@ export function EditableContact({
                 </Content>
               </div>
             ) : null}
+            {!userContact.telephoneNumber && companyProfile.telephoneNumber ? (
+              <div>
+                <Caption>Company Phone</Caption>
+                <Content>
+                  <p>{companyProfile?.telephoneNumber}</p>
+                </Content>
+              </div>
+            ) : null}
```

Comment:
To avoid showing Telephone Number twice in Job Listing pages.

Reply:
Oh ok, we have `userContact.telephoneNumber` AND `companyProfile.telephoneNumber`.